src/stcal/ramp_fitting/src/slope_fitter.c

```diff
@@ -784,10 +850,55 @@ print_pid_info(long long prev, int line, char * label) {
 }
 
 
+/* ------------------------------------------------------------------------- */
+/*                              PROC LOGGER                                  */
+/* ------------------------------------------------------------------------- */
+
+/*
+ * This logging system is for debugging purposes.  The 'log_dir' variable needs to
+ * be changed to suit the local directory structure.  Even if fopen fails, there are
+ * checkers in place such that the logger is simply ignored.  Maybe a more intelligent
+ * system should be put in place to create a logger directory, but for now this
+ * suffices.
+ * 
+ * I created this to debug multirpocessing.  The logger captures process information
+ * into separate logs, so any failures can be determined on a per process bases.
+ * Redirection from the command line behaves unexpectedly when multiple processes print
+ * to the terminal and copy and pasting from the terminal is lacking due to the amount
+ * of data that possibly needs to be copied.
+ * 
+ * Each process has its own log and each log has a timestamp, so if you have successive
+ * runs, each can be separated by time, as well as process ID.
+ */
+void
+set_up_logger() {
+    const char * log_dir = "/Users/kmacdonald/code/stcal/logs";
```

Comment:
Maybe get this location from an environment variable or some other way?

Reply:
It's a debugging function that isn't used anywhere.  I left it there so I can use it in the future for debugging.  I'll simply remove the path.